pilot/pkg/model/jwks_resolver.go

```diff
@@ -212,8 +212,10 @@ func newJwksResolverWithCABundlePaths(
 				Proxy:             http.ProxyFromEnvironment,
 				DisableKeepAlives: true,
 				TLSClientConfig: &tls.Config{
-					RootCAs:    caCertPool,
-					MinVersion: tls.VersionTLS12,
+					// nolint: gosec
```

Comment:
nit: any nolint:gosec should explain _why_. In this case its simple - user explicitly opted into insecure.